Actions/Sign/Sign.ps1

```diff
@@ -12,13 +12,12 @@ param(
 )
 
 $telemetryScope = $null
-$bcContainerHelperPath = $null
 
 try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
     Import-Module (Join-Path -path $PSScriptRoot -ChildPath "..\TelemetryHelper.psm1" -Resolve)
     Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "Sign.psm1" -Resolve)
-    $BcContainerHelperPath = DownloadAndImportBcContainerHelper -baseFolder $ENV:GITHUB_WORKSPACE
+    DownloadAndImportBcContainerHelper -baseFolder $ENV:GITHUB_WORKSPACE
```

Comment:
Same here: BCContainerHelper shouldn't be a requirement for signing.

Reply:
The telemetry pieces are built on functionality in containerHelper (nesting all telemetry within a workflow).
We might want to refactor the telemetry at some point in time to not use containerhelper, but that is another story.